ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/ethernet-interfaces.j2

```diff
@@ -12,7 +12,7 @@ ethernet_interfaces:
 {%             set adapter_settings = {} | combine(parent_profile | arista.avd.default({}),
                                                    adapter_profile | arista.avd.default({}),
                                                    adapter, recursive=true, list_merge='replace') %}
-{%             set endpoint_ports = adapter.endpoint_ports | arista.avd.default(adapter.server_ports) %}
+{%             set endpoint_ports = adapter_settings.endpoint_ports | arista.avd.default(adapter_settings.server_ports) %}
 {%             set peer_type = connected_endpoints_keys[endpoint_key].type %}
 {%             set peer = connected_endpoint %}
 {%             for adapter_switch in adapter.switches %}
```

Comment:
@ClausHolbechArista, I think we can modify this parameter, ```adapter.switches``` to ```adapter_settings.switches```. This gives the user a lot of flexibility. #e.g. 20 end_points connected to the same group of switches. So if we add this to profile settings, the user can use the profile settings. Anyway we have the local settings to override the default ones.

Reply:
I added a third commit to fix the switch side as well, in case someone wanted to have port profile strides in that dimension instead of in the server interface dimension like I did.